README.md

```diff
@@ -54,23 +29,20 @@ After you're finished please remove all the comments and instructions!
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
 # 📖 [your_project_name] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+
+**[To do list ]** is a small app that's add and remove books 
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
```

Comment:
- [ ] As mentioned above please fix the tech stack sections to include the right stack currently it is :point_down: 
![image](https://user-images.githubusercontent.com/97521557/214374567-53ecd6fd-2efa-4522-91c2-d075562b03f0.png)